packages/backend/migrations/20240827115832_update_resources_with_tenantId.js

```diff
@@ -20,10 +20,6 @@ exports.up = function (knex) {
       table.uuid('tenantId').notNullable()
       table.foreign('tenantId').references('id').inTable('tenants')
     })
-    .table('grants', function (table) {
```

Comment:
Grant's don't need `tenantId` - they're never queried on the backend by that filter, and the tenant it is for can be determined by the auth server URL path. Having the requirement explicitly in this table introduces some backwards compatibility issues.